rclcpp/README.md

```diff
@@ -2,6 +2,8 @@
 
 The ROS client library in C++. Visit the [rclcpp API documentation](http://docs.ros2.org/latest/api/rclcpp/) for a complete list of its main components.
```

Comment:
I don't think it make sense to link to the same page twice in two paragraphs. Why don't you just combine them into a single paragraph stating that the feature documentation and API documentation can both be found there.